ide/api/project.py

```diff
@@ -280,6 +283,44 @@ def begin_export(request, project_id):
 
 
 @login_required
+@require_safe
+@json_view
+def get_projects(request):
```

Comment:
I suppose it's a fairly ambiguously named function. I added it for the dependencies screen, and it's used to set up the list of your cloudpebble libraries which your project can depend on. I'll add a comment of some sort to make it more clear.